nixos/modules/system/boot/systemd/journald.nix

```diff
@@ -96,6 +96,7 @@ in {
       "systemd-journald@.service"
       "systemd-journal-flush.service"
       "systemd-journal-catalog-update.service"
+      "systemd-journald-sync@.service"
```

Comment:
No specific docs; main docs are https://www.freedesktop.org/software/systemd/man/latest/systemd-journald.service.html

Source is https://github.com/systemd/systemd/blob/main/units/systemd-journald-sync%40.service